package.json

```diff
@@ -15,10 +15,15 @@
     "@mui/icons-material": "^5.15.4",
     "@mui/material": "^5.15.4",
     "axios": "^1.6.4",
+    "localforage": "^1.10.0",
+    "match-sorter": "^6.3.3",
     "material-ui-popup-state": "^5.0.10",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-material-ui-carousel": "^3.4.2",
+    "react-router": "^6.21.3",
```

Comment:
If you don't need to use react-native things then react-router-dom is good enough.

Reply:
maybe it's my mistake, but I don't see myself using a react router